File: PY/Bryann Valderrama/Matemática/GCD_LCM.py
from sys import stdout


def GCD(a, b):
    return a if b == 0 else GCD(b, a % b)


def LCM(a, b):
    return a * (b // GCD(a, b))


a = 8
b = 53
stdout.write(f'GCD de a y b = {GCD(a,b)}\n')
stdout.write(f'LCM de a y b = {LCM(a,b)}')

File: C++/Bryann Valderrama/Matemática/GCD_LCM.cpp
#include <cstdlib>
#include <bits/stdc++.h>
using namespace std;

int GCD(int a, int b)
{
    return b == 0 ? a : GCD(b, a % b);
}

int LCM(int a, int b)
{
    return a * (b / GCD(a, b));
}

int main(int argc, char const *argv[])
{
    int a = 8;
    int b = 53;
    cout << "GCD de a y b = " << GCD(a, b) << endl;
    cout << "LCM de a y b = " << LCM(a, b) << endl;
    return 0;
}
